Izebiafe/Blog
# Blog app - Controllers

# Pull Request: Project Initialization and Database Setup

## Changes Made

- Created a new app named Blog.
- Initialized the project with Git.
- Ensured that the project is set up to use a Postgres database.
- Built the initial schema for the project.

## Instructions

1. **Create a New App:**
   - Created a new app with a meaningful name, ensuring it reflects the purpose of the project.

2. **Git Initialization:**
   - Initialized the project with Git for version control.

3. **Postgres Database Setup:**
   - Ensured that the project is configured to use a Postgres database.

4. **Schema Building:**
   - Built the initial schema for the project, laying the foundation for future development.

## Additional Information

- Branch Name: initialbranch
- Database: Postgres

## Testing Instructions

- Ensure that the app runs without errors.
- Verify that the database connections are correctly configured.
- Check the schema to confirm that it aligns with the project requirements.

## Reviewer's Note

Please review the changes made to ensure that the app is correctly set up and the initial schema is in line with the project's objectives.
